### DISCUSSION TITLE
这个设计器  有没有vue版本的   现有的改成vue版本的困难吗

### DISCUSSION THREAD

**Discussion by magicalStan** [alibaba/designable#25] [Q&A]
这个设计器  有没有vue版本的   现有的改成vue版本的困难吗

**Comment by xjchenhao** [comment]
> 这个设计器 有没有vue版本的 现有的改成vue版本的困难吗

建议当作独立的工具去开发，不要被框架限制自己的成长。

**Comment by borina555** [comment]
设计器用React版本， 生成schema数据用vue接收， 可行吗？  我测试发现一些常用的基础输入组件是可以的， 但是 FormTab等组件有问题
